tests/compiler/circuits/duplicate_name_context.leo

```diff
@@ -1,6 +1,6 @@
 /*
 namespace: Compile
-expectation: Pass
+expectation: Fail
```

Comment:
@acoglio @collinc97 @bendyarm Double checking that this test should be changed to `Fail`, per our shadowing rules.

Reply:
Yes that test should be changed to `Fail`